bokehjs/src/lib/models/annotations/tooltip.ts

```diff
@@ -112,7 +114,8 @@ export class TooltipView extends AnnotationView {
     // be problematic
     if (this.el.childNodes.length > 0) {
       this.el.style.top = `${top}px`
-      this.el.style.left = `${left}px`
+      this.el.style.left = left ? `${left}px` : 'unset'
+      this.el.style.right = right ? `${right}px` : 'unset'
```

Comment:
`unset` in not supported by IE.

Reply:
Oh! I'll switch it to `inherit` I guess.